_posts/2015-06-16-hard-fork-policy.md

```diff
@@ -0,0 +1,41 @@
+---
+type: posts
+layout: post
+lang: en
+category: blog
+
+title: "Bitcoin.org Hard Fork Policy"
+permalink: /en/posts/hard-fork-policy.html
+date: 2015-06-16
+---
+It appears that the recent block size debate will likely result in a
+contentious hard fork attempt.
+
+Contentious hard forks are bad for Bitcoin. At the very best, a
+contentious hard fork will leave people who chose the losing side of the
+fork feeling disenfranchised. At the very worst, it will make bitcoins
+permanently lose their value. In between are many possible outcomes, but
+none of them is good.
```

Comment:
Thanks to you and psztorc for pointing that out.  I'll make that change and @morcos's proposed change in a little bit.